book.rb

```diff
@@ -0,0 +1,13 @@
+class Book
+  attr_accessor :title, :author
```

Comment:
- [ ] I am requesting some of these changes because they were introduced after the previous review. Kindly add `:rentals` to this attr_accessor so we can use it in the rental class.